pkgs/development/python-modules/solaredge/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, pythonOlder
+, pytz
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "solaredge";
+  version = "0.0.4";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q4gib0l3qnlpncg84ki027vr1apjlr47vd6845rpk7zkm8lqgfz";
+  };
+
+  propagatedBuildInputs = [
+    python-dateutil
```

Comment:
Where did you find this dependency? I can't find it in the *only* commit in this repo: https://github.com/bertouttier/solaredge/commit/2c94b38527374b0abd088e2819455b332bc5153a.patch.

Reply:
Bah, the git repo is out of date.